src/main/java/com/linkedin/kmf/services/SignalFxMetricsReporterService.java

```diff
@@ -155,7 +151,7 @@ private void captureMetrics() {
       List<MbeanAttributeValue> attributeValues = getMBeanAttributeValues(mbeanExpr, attributeExpr);
 
       for (final MbeanAttributeValue attributeValue : attributeValues) {
-        String metric = attributeValue.toString();
+        String metric = attributeValue.toString();        
```

Comment:
Remove trailing whitespace.

Also, all this code (lines 154-162) is kinda weird. What's the "shape" of what you're working from, and what are you trying to get to?